.travis.yml

```diff
@@ -67,11 +67,11 @@ install: |
 
 
 script:
-  - if [ "$TASK" = "core-tests"    ]; then $ANT core-tests    && $ANT serialization-tests && $ANT coverage; fi
+  - if [ "$TASK" = "core-tests"    ]; then $ANT core-tests    && $ANT coverage; fi
   - if [ "$TASK" = "checkstyle"    ]; then $ANT checkstyle    && $ANT eclipseformat-check && run cd tools/kompos && nvm install 7 && run npm install && run npm run lint; fi
   - if [ "$TASK" = "kompos-tests"  ]; then run nvm install 8  && $ANT && cd tools/kompos  && run npm -s run verify && run npm test; fi
   - if [ "$TASK" = "replay1-tests" ]; then $ANT compile       && run ./tests/replay/test.sh 1; fi
   - if [ "$TASK" = "replay2-tests" ]; then $ANT compile       && run ./tests/replay/test.sh 2; fi
-  - if [ "$TASK" = "snapshot-tests" ]; then $ANT compile      && run ./tests/snapshot/test.sh; fi
+  #- if [ "$TASK" = "snapshot-tests" ]; then $ANT compile      && run ./tests/snapshot/test.sh; && $ANT serialization-tests fi
```

Comment:
I guess this doesn't strictly need to be commented out anymore.
If so, perhaps a brief note on why commented out, please. Makes it easier to track this.

Reply:
uncommented it and added a comment to the other commented thing

Reply:
hm looks like this task gets executed if i uncomment this line...

Reply:
looks more like a [syntax error](https://travis-ci.org/smarr/SOMns/jobs/568721883#L2019) to me:

```
/home/travis/.travis/functions: eval: line 104: syntax error near unexpected token `&&'
/home/travis/.travis/functions: eval: line 104: `if [ "$TASK" = "snapshot-tests" ]; then $ANT compile      && run ./tests/snapshot/test.sh; && $ANT serialization-tests fi '
```

I would guess the `;` before the `&&` should be removed.